packages/dev/core/src/Shaders/pbr.fragment.fx

```diff
@@ -370,6 +371,43 @@ void main(void) {
         #endif
     #endif
 
+    // _____________________________ Iridescence ____________________________
+    #ifdef IRIDESCENCE
+        iridescenceOutParams iridescenceOut;
+
+        #ifdef IRIDESCENCE_TEXTURE
+            vec2 iridescenceMapData = texture2D(iridescenceSampler, vIridescenceUV + uvOffset).rg * vIridescenceInfos.y;
+        #endif
+        #ifdef IRIDESCENCE_THICKNESS_TEXTURE
+            vec2 iridescenceThicknessMapData = texture2D(iridescenceThicknessSampler, vIridescenceThicknessUV + uvOffset).rg * vIridescenceInfos.w;
+        #endif
+
+        iridescenceBlock(
+            vIridescenceParams,
+            #ifdef IRIDESCENCE_TEXTURE
+                iridescenceMapData,
+            #endif
+            #ifdef IRIDESCENCE_THICKNESS_TEXTURE
+                iridescenceThicknessMapData,
+            #endif
+            iridescenceOut
+        );
+
+        float topIor = 1.; // Assume air
+        float viewAngle = NdotV;
+        float iridescenceIntensity = iridescenceOut.iridescenceIntensity;
+
+        #ifdef CLEARCOAT_IRIDESCENCE_REMAP_TODO

```

Comment:
As I understand it, for the time being `CLEARCOAT_IRIDESCENCE_REMAP_TODO` is never defined by the js code so we never run this block of code (hence the `TODO` in the name)?

Reply:
Yup I need to do in a subsequent PR :-) just here as a reminder

Reply:
I actually implemented it :-)